verticapy/utilities.py

```diff
@@ -485,19 +483,27 @@ def pandas_to_vertica(
 read_csv  : Ingests a CSV file into the Vertica database.
 read_json : Ingests a JSON file into the Vertica database.
 	"""
+    check_types(
+        [
+            ("name", name, [str],),
+            ("schema", schema, [str],),
+        ]
+    )
     cursor, conn = check_cursor(cursor)[0:2]
-    name = "verticapy_df_{}".format(random.randint(10e5, 10e6))
+    if not(name):
+        name = "verticapy_df_{}".format(random.randint(10e5, 10e6))
     path = "{}.csv".format(name)
     try:
         df.to_csv(path, index=False)
-        read_csv(path, cursor, table_name=name, temporary_local_table=True, parse_n_lines=10000)
+        if not(name):
+            vdf = read_csv(path, cursor, table_name=name, temporary_local_table=True, parse_n_lines=10000)
+        else:
+            vdf = read_csv(path, cursor, table_name=name, schema=schema, temporary_local_table=False, parse_n_lines=10000)
```

Comment:
I suppose you have picked `parse_n_lines=10000` based on practical reasons. Wouldn't it be better if you make it an optional input parameter of the function with default value of 10000?

Reply:
Done;